import-export-cli/cmd/deleteAPIPolicy.go

```diff
@@ -0,0 +1,87 @@
+/*
+*  Copyright (c) WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Shall we update the license header with the year and Inc to LLC?

Reply:
Do we have an example license header?